apps/web/src/pages/Applications/ApplicationCareerTimelineAddPage/components/AddExperienceForm.tsx

```diff
@@ -65,7 +65,7 @@ const AddExperienceForm = ({ applicationId }: AddExperienceFormProps) => {
     formValues,
   ) => {
     const submitData = formValuesToSubmitData(formValues, [], type);
-    const args = getMutationArgs(userAuthInfo?.id || "", submitData);
+    const args = getMutationArgs(userAuthInfo?.id ?? "", submitData);
```

Comment:
Another pre-existing, but why would we pass an empty string into this function?  Shouldn't we error out if there's no user ID?